client/src/main/scala/com/dwolla/cloudflare/domain/model/accesscontrolrules/RuleScope.scala

```diff
@@ -0,0 +1,5 @@
+package com.dwolla.cloudflare.domain.model.accesscontrolrules
+
+case class RuleScope(id: String,
```

Comment:
Should this ID be a tagged string?

Reply:
To stay consistent, it probably should. `RuleScope`'s id is the rule's associated user's id:
```"scope": {
        "id": "7c5dae5552338874e5053f2534d2767a",
        "email": "user@example.com",
        "type": "user"
      }```

Reply:
Going to leave this as a `String` for now while we do some more research. We'll update it in the next milestone.